### DISCUSSION TITLE
Media control buttons for "go to start" and "go to end"

### DISCUSSION THREAD

**Discussion by me-nkr** [motion-canvas/motion-canvas#811] [Ideas]
What the title says, I am aware of the fact that you use timeline for this, but these buttons could be useful I think.

Suggestion: we could use these buttons⏮and⏭respectively for start and end.

**Comment by aarthificial** [comment]
It's a good idea. Feel free to [create a feature request](https://github.com/motion-canvas/motion-canvas/issues/new/choose) for it in the main repo.

Semi-related: you can use `SHIFT + ArrowLeft` and `SHIFT + ArrowRight` to go to the start and end respectively.
